server-side/models/invitation.model.js

```diff
@@ -0,0 +1,10 @@
+import mongoose,{Schema} from 'mongoose';
+
+const InvitationsSchema = new mongoose.Schema({
+    invitedUserID: { type: Schema.Types.ObjectId, ref: 'Users', required: true },
```

Comment:
missing inviterID.
notice that invitedUserID could be user or pendingUser... think about solutions